### PR TITLE
Responsive nav

### PR DESCRIPTION
**Pull request by ishimwezachee**
# Responsive nav
 In this project I am supposed to do the following 

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (SajjadAhmad14) [inline .eslintrc.json:1]: Please use linter for HTML, CSS, and JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.

File: index.html [hunk 2]
```diff
@@ -31,19 +31,30 @@
...
             <i class="far fa-bars"></i>
           </a>
         </nav>
       </div>
       <div class="desktop-nav">
         <ul class="nav-list">
-          <li class="top-nav">Portoflio</li>
-          <li class="top-nav">About</li>
-          <li class="top-nav">Contact</li>
+          <li class="close-menu">
```
Comment (SajjadAhmad14) [inline index.html:44]: Please use font for the hamburger items from the design page.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Zachee Ishimwe, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mke2111 — APPROVED** [review]
### Hi @ishimwezachee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**